### PR TITLE
fix: TT-25 Empty selection in Activity drop-down(Time Clock page)

### PR DESCRIPTION
**Pull request by lenshinoda**
Hi Team, I solved the problem by checking if the activity is not empty in the  onSubmit() method.

### CODE REVIEW

File: src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts [hunk 1]
```diff
@@ -466,6 +467,15 @@ describe('EntryFieldsComponent', () => {
       expect(featureToggleGeneralService.isActivated).toHaveBeenCalled();
     });
   });
+
+  it('toastrService should be called when the activity field is empty', () => {
+    fixture.detectChanges();
+    const select: HTMLSelectElement = fixture.debugElement.query(By.css('select')).nativeElement;
```
Comment (scastillo-jp) [inline src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts:473]: add a line white after this line

File: src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts [hunk 2]
```diff
@@ -133,6 +133,10 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
   }
 
   onSubmit() {
+    if (this.entryForm.value.activity_id === '-1') {
```
Comment (scastillo-jp) [inline src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts:136]: The empty option has an id **-1** ?

File: src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts [hunk 3]
```diff
@@ -466,6 +467,15 @@ describe('EntryFieldsComponent', () => {
...
+    spyOn(toastrServiceStub, 'error');
+    select.value = select.options[0].value;
+    select.dispatchEvent(new Event('change'));
```
Comment (scastillo-jp) [inline src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts:476]: add a line white after this line

**Review by scastillo-jp — APPROVED** [review]

File: src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts (hunk 2 above)
Comment (lenshinoda) [reply src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts:136]: Yes, it was done in an old PR to solve another problem: https://github.com/ioet/time-tracker-ui/blob/10e7694a56bd6cca2df98664d54175bdfa293f86/src/app/modules/time-clock/components/entry-fields/entry-fields.component.html

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**